### PR TITLE
Checker: Add time range to report

### PR DESCRIPTION
**Pull request by s-l-teichmann**
Part of #394

### CODE REVIEW

**Review by JanHoefelmeyer — APPROVED** [review]
LGTM & works

**Review by JanHoefelmeyer — APPROVED** [review]
LGTM & works

File: docs/csaf_checker.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 Usage:
   csaf_checker [OPTIONS] domain...
 
+Application Options:
```
Comment (bernhardreiter) [inline docs/csaf_checker.md:9]: @JanHoefelmeyer adding trailing newlines should be avoided. (We can address this in a cleanup pr.)

**Review by bernhardreiter — APPROVED** [review]
LGTM

**Pull request merged**